packages/platform-server/test/hydration_spec.ts

```diff
@@ -81,9 +81,11 @@ function verifyClientAndSSRContentsMatch(ssrContents: string, clientAppRootEleme
  * Walks over DOM nodes starting from a given node and checks
  * whether all nodes were claimed for hydration, i.e. annotated
  * with a special monkey-patched flag (which is added in dev mode
- * only).
+ * only). It skips any nodes with the skip hydration attribute.
  */
 function verifyAllNodesClaimedForHydration(el: HTMLElement) {
+  if ((el.nodeType === Node.ELEMENT_NODE && el.hasAttribute('ngskiphydration'))) return;
```

Comment:
Let's may be create a local const in this file with the name (so it'd be easier to find later if needed).
```suggestion
  if ((el.nodeType === Node.ELEMENT_NODE && el.hasAttribute(SKIP_HYDRATION_ATTR_NAME))) return;
```

Reply:
I think there were issues with importing this symbol here, but I'll try again to see if it's safe to do.